1-data-marketplace/infra/analytics/co-managed/main.bicep

```
@@ -186,3 +186,4 @@ output serviceProviderDatalakeName string = serviveProviderDatalake.outputs.name
 output serviceProviderDatalakeResourceId string = serviveProviderDatalake.outputs.id
 output resourceGroupName string = useExistingCoManagedResourceGroup ? analyticsExistingCoManagedRg.name : analyticsCoManagedRg.name
 output keyVaultName string = keyvault.outputs.keyVaultName
+output purviewResourceName string = (useExistingPurview) ? purview.outputs.purviewName : ''
```

we should output the name of the purview to be consistent and not to have to handle the empty string special case

```suggestion
output purviewResourceName string = useExistingPurview ? purview.outputs.purviewName : last(split(purviewResourceId, '/'))
```

unfortunately, bicep doesn't provide a first class func to extract the resource name from its ID, so using `split() `